config.json

```diff
@@ -56,7 +56,8 @@
     "dot-dsl",
     "custom-set",
     "nth-prime",
-    "forth"
+    "forth",
+    "beer-song"
```

Comment:
Yeah, the `config.json` determines the order in which people receive exercises by default. (We really need to make that easier to understand/discover!)